app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OrdersController < ApplicationController
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @order = Order.find(params[:id])
+  end
+
+  def new
+    @order = Order.new
+  end
+
+  def create
+    @order = current_user.orders.new(order_params)
+    @order.cart_items << current_cart.cart_items
```

Comment:
again, this will very much be changed after cart changes, but anyway.
This is way too much for an action.
When you have even a little more complex logic than just `save`, please consider using services